.github/ISSUE_TEMPLATE/oss_contribution.yml

```
@@ -0,0 +1,57 @@
+name: OSS Contribution
```

```suggestion
name: Data Contribution
```
What about modifying the name to "Data Contribution"?

---

Done